.github/workflows/ci_pkg-install.yml

```diff
@@ -8,8 +8,12 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master]
 
 jobs:
-
   pkg-install:
+    if: |
```

Comment:
this is very fast test and also some docs may interfere with packaging...

Reply:
alright.